package.json

```diff
@@ -30,6 +30,7 @@
   "dependencies": {
     "async": "^2.5.0",
     "debug": "^2.6.0",
+    "moment": "^2.18.1",
```

Comment:
Including moment for those two calls seems a little bit much -- is there a good reason?

Reply:
Agreed. I've removed that and force pushed.